modules/ROOT/pages/design-modify-raml-specs-conform.adoc

```diff
@@ -0,0 +1,791 @@
+= Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0
+
+[[bookmark-a,Back to the top]]
+
+As of November 1, 2018, the API editor Design Center is now more precise in ensuring that the RAML in your API specifications is valid. However, it is possible that one or more of the APIs that you used previous versions of the API editor to create and publish to Anypoint Exchange includes invalid RAML that those previous versions did not flag.
```

Comment:
I'm thinking if we don't need to write "Anypoint Design Center", we can also omit the "Anypoint" from Exchange.

```suggestion
As of November 1, 2018, the API editor in Design Center is now more precise in ensuring that the RAML in your API specifications is valid. However, it is possible that one or more of the APIs that you used previous versions of the API editor to create and publish to Exchange includes invalid RAML that those previous versions did not flag.
```